tests/test_wog.py

```diff
@@ -18,3 +18,28 @@ def test_silithid_swarmer():
 	assert game.player2.hero.health == 30 - 1 - 3
 	assert not silithid.cant_attack
 	assert not silithid.can_attack()
+
+	
+def test_corrupted_healbot():
+	game = prepare_game()
+	hb1 = game.player1.give("OG_147")
+	hb1.play()
+	hb2 = game.player1.give("OG_147")
+	hb2.play()
+	game.end_turn()
+	game.end_turn()
+	hb3 = game.player1.give("OG_147")
+	hb3.play()
+	hb4 = game.player1.give("OG_147")
+	hb4.play()
+
+	game.player2.hero.set_current_health(13)
+	assert game.player2.hero.health == 13
+	hb1.destroy()
+	assert game.player2.hero.health == 13 + 8
+	hb2.destroy()
+	assert game.player2.hero.health == 13 + 8 + 8
+	hb3.destroy()
+	assert game.player2.hero.health == 30
+	hb4.destroy()
+	assert game.player2.hero.health == 30
```

Comment:
This test is too extensive for a card that is already tested (cf `test_zombie_chow`). It also doesn't respect the common test style.

You can safely drop it imo.